### PR TITLE
QA SP update 3

### LINKED ISSUE #329: QA Standards Update 3
QA is now requesting the scope of work and a list of known issues before the kickoff meeting. We are also emphasizing the importance of an acceptance test and clarifying that the developers will create the project boards and milestones with the definition of done. QA will assist and guide the developers through those if needed during the kickoff meeting.

### PR DESCRIPTION
**Pull request by dcedano**
QA is now requesting the scope of work and a list of known issues before the kickoff meeting. We are also emphasizing the importance of an acceptance test and clarifying that the developers will create the project boards and milestones with the definition of done. QA will assist and guide the developers through those if needed during the kickoff meeting.

## Changes
_List Changes Introduced by this PR_
1. PM will provide Scope of Work and List of Know Issues when scheduling QA
2. Expressing the importance of an acceptance test to encourage the developers to do it and reach out to QA for assistance 
3. Clarifying that the developers will be the ones to create and manage the projects boards, and create the milestone for this sprint with the definition of done. If the developer does not do this before the kickoff meeting, QA will step in to assist and guide the developer. 

## Purpose
_Clarification for QA prior to the sprint and clarification of duties for QA and developers._

## Approach
_Adding the SOW and LOK as requirements when scheduling and being more specific with what we were already requesting._

Closes #329

### CODE REVIEW

**Review by mwallert — APPROVED** [review]
Changes look good to me!

**Comment by dcedano** [comment]
thanks @mwallert !!! I was getting nervous 😅

File: standards/qa-process.md [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
   - Delivery/testing timeframe
-  - Associated milestone
+  - Associated milestone containing Definition of Done
   - Associated project board
   - Sandbox URL/ Testflight App Name
   - App Version number(s) to be tested
   - Devices / Browsers to be tested
   - Definition of Done
-  - Acceptance Test in Markdown Format.
+  - Acceptance test in Markdown Format. An acceptance test will be necessary for all new projects. If you would like a tutorial on creating an acceptance test please contact qa@bitwiseindustries.com and we can set up a meeting to get you started.
```
Comment (michaelachrisco) [inline standards/qa-process.md:28]: I would suggest creating an acceptance test defined on another page to reduce the number of requests QA gets in the long run. 

For now this is not a blocker, but should probably be on our radar. @dcedano do you know if we already have docs in place for acceptance testing?

**Review by michaelachrisco — APPROVED** [review]
Looks good. I like the more defined roles of each of the different departments and constructive info for everyone involved. One non-blocking question.

**Pull request merged**